x/rollup/keeper/msg_server.go

```diff
@@ -239,7 +244,7 @@ func ValidateBasic(m *rollupv1.InitiateWithdrawalRequest) error {
 	}
 	// Check if the gas limit is within the allowed range.
 	gasLimit := new(big.Int).SetBytes(m.GasLimit).Uint64() // size=24 (0x18), offset=64 (0x40)
```

Comment:
Does the `// size=24 (0x18), offset=64 (0x40)` comment apply here? Could it be removed?